pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -444,6 +444,14 @@ def apply_operations(self, operations):
             elif isinstance(operations[0], BasisState):
                 self._apply_basis_state(operations[0].parameters[0], operations[0].wires)
                 operations = operations[1:]
+            elif isinstance(operations[0], MPSPrep):
+                if self.method == "tn":
```

Comment:
Do we want to do two `if` checks here?

Reply:
Yes, because, just like `StatePrep,` the method 'tn' doesn't support `MPSPrep` due to the limitation of the max bond dimension of 2. Your concert might be related to performance, but usually, we call this gate only once, such as `StatePrep`.
Additionally, for the method `tn`, `StatePrep` and `MPSPrep` will be equivalent.

Reply:
It's more  likely the function will go to the second branch right?

Reply:
Done!